python/ray/tune/tests/test_trial_runner.py

```diff
@@ -19,6 +19,9 @@
 
 
 class TrialRunnerTest(unittest.TestCase):
+    def setUp(self):
+        ray.shutdown()
```

Comment:
This seems like something that should belong in a `tearDown` rather than a `setUp`, right?

Reply:
Like, this seems to suggest that some other test isn't getting torn down properly.

Reply:
good point... will close for now